psi4/broker.py

```diff
@@ -0,0 +1,132 @@
+# Copyright (c) 2013-2018 Thomas Spura <thomas.spura@gmail.com>
```

Comment:
For files within Psi4 you will need to use the Psi4 copyright header found in all other Python files.